src/primitives/useMouse.ts

```diff
@@ -61,15 +61,56 @@ const handleClick = (e: MouseEvent): void => {
       (active.height || 0) * precision,
     )
   ) {
-    document.dispatchEvent(createKeyboardEvent('Enter', 13));
-    setTimeout(
-      () =>
-        document.body.dispatchEvent(createKeyboardEvent('Enter', 13, 'keyup')),
-      1,
-    );
+    if (active.onMouseClick) {
+      sendMouseClickEvent(e, active);
```

Comment:
are there benefits to having a separate function? Do you have cases where you want mouse click to do a different action?